starboard/loader_app/BUILD.gn

```diff
@@ -54,10 +54,9 @@ if (sb_is_evergreen_compatible && sb_evergreen_compatible_package) {
   copy("copy_loader_app_lib") {
     install_content = true
     if (target_cpu == "arm" && arm_float_abi == "softfp") {
-      sources = [ "$root_out_dir/../evergreen-$target_cpu-${arm_float_abi}_$build_type/libcobalt.so" ]
+      sources = [ "$root_out_dir/../evergreen-$target_cpu-${arm_float_abi}_$build_type/install/lib/libcobalt.so" ]
```

Comment:
If libcobalt is stripped, the stack trace of crashes on buildbot or locally will not contain symbols, and we'll have to go through the steps to symbolize them. Note that partners are already using the stripped libcobalt which is downloaded from Github.

Reply:
That's true, but for automated testing I believe we're currently installing the stripped cobalt shared library on the device for existing evergreen-compatible platforms (e.g., see https://github.com/youtube/cobalt/blob/main/starboard/evergreen/shared/launcher.py#L265 for raspi). So I thought this change would make things more consistent, and also maybe avoid partner confusion about why the APK is so much bigger when using a locally built libcobalt vs. one downloaded from GitHub.

Reply:
Ok. Sgtm. Let's make the change and see if the potential issue bothers us for debugging.

Reply:
Sounds good, thanks.